packages/@aws-cdk/aws-codebuild/lib/project.ts

```diff
@@ -1036,6 +1045,53 @@ export class Project extends ProjectBase {
     };
   }
 
+  private renderLogs(props: ProjectProps): CfnProject.LogsConfigProperty {
```

Comment:
Can we make this method take `LoggingOptions` instead? We don't need the entire `ProjectProps` interface here.